en/theme/material/base.html

```diff
@@ -74,9 +74,9 @@
     {% endif %}
     {% block extrahead %}{% endblock %}
     <!-- CookiePro Cookies Consent Notice start for wso2.com -->
-    <script type="text/javascript" src="https://cookie-cdn.cookiepro.com/consent/486163bc-a8c5-40d8-b185-c707cc718a23/OtAutoBlock.js" ></script>
+    <!-- <script type="text/javascript" src="https://cookie-cdn.cookiepro.com/consent/486163bc-a8c5-40d8-b185-c707cc718a23/OtAutoBlock.js" ></script>
     <script src="https://cookie-cdn.cookiepro.com/scripttemplates/otSDKStub.js"  type="text/javascript" charset="UTF-8" data-domain-script="486163bc-a8c5-40d8-b185-c707cc718a23" ></script>
-    <script src="https://wso2.com//sites/all/themes/wso2_d7/js/cookie-revoke.js"></script>
+    <script src="https://wso2.com//sites/all/themes/wso2_d7/js/cookie-revoke.js"></script> -->
```

Comment:
Why this is commented out?

Reply:
Commented the Cookie Pro pop-up to ease testing. Uncommented now.